test_requirements.txt

```diff
@@ -1,6 +1,6 @@
 tox >= 3.23.0
 pygments >= 2.8.0
-jinja2 == 3.1.3
+jinja2 == 3.1.4
```

Comment:
this would impact the web GUI henceforth needs to be validated on live platform before upgrade.